fairlearn/datasets/__init__.py

```diff
@@ -8,9 +8,11 @@
 from ._fetch_adult import fetch_adult
 from ._fetch_boston import fetch_boston
 from ._fetch_bank_marketing import fetch_bank_marketing
+from .synthetic import make_synthetic_dataset
```

Comment:
I can move this to `_synthetic`.

Reply:
Good point! This is mostly to show everything at the module level (`from fairlearn.datasets import make_synthetic_dataset` in the users' code) rather than having people import `fairlearn.datasets.synthetic`.